OracleDataIntegrator/dockerfiles/12.2.1.2.6/oraInst.loc

```diff
@@ -0,0 +1,2 @@
+inventory_loc=/u01/oracle/.inventory
```

Comment:
I'm curious: is there a particular reason why you're hiding the inventory location with a dotfile? I've always put the inventory in `/u01/app/oracle/oraInventory` when installing Oracle products.

Note this is not a merge blocker, I'm just wanted to learn more. :)